packages/deployment/lib/kes.js

```diff
@@ -166,6 +166,20 @@ class UpdatedKes extends Kes {
     });
   }
 
+  /**
+  * Build list of buckets of desired type.
+  *
+  * @param {Object} buckets - config buckets
+  * @param {any} bucketType - selected type.
```

Comment:
This is a `string`, right?

Reply:
Yes, I keep forgetting that when I use my js-doc-template.